core/plugin.go

```diff
@@ -85,6 +85,16 @@ var (
 		metricPluginsStopping,
 		metricPluginsDead,
 	}
+
+	// Has to be index parallel to PluginState*
+	stateToDescription = []string{
```

Comment:
It might be a better idea to solve this translations with struct tags (https://golang.org/pkg/reflect/#StructTag).
Programatically more complex to resolve but avoids having to change two places in a file where the second part may be forgotten.